bin/run.js

```diff
@@ -23,4 +23,9 @@ import { createErrorPromise } from "../dist/countlyInterceptor.js";
 oclif
   .run(process.argv.slice(2), import.meta.url)
   .then(oclif.flush)
-  .catch(/** @param {unknown} error */ (error) => createErrorPromise(error));
+  .catch(
+    /** @param {unknown} error */ (error) => {
+      console.error(error);
```

Comment:
It's weird for me that error doesn't still fucking print. We can't do it like you did it here because this will show stack traces for ALL errors, which we don't want to do. I will try to reproduce this and figure out the problem and solve it correctly

Reply:
fixed this correctly, I think